src/pybind/mgr/dashboard_v2/controllers/rbd.py

```diff
@@ -68,3 +84,29 @@ def get(self, pool_name):
         if status == ViewCache.VALUE_EXCEPTION:
             raise value
         return {'status': status, 'value': value}
+
+    def create(self, data):
+        if not self.rbd:
+            self.rbd = rbd.RBD()
+
+        # Get input values
+        size = data.get('size', 4 * 1024 ** 3)
+        obj_size = data.get('obj_size', 2 ** 22)
+        features = data.get('features', [])
+        stripe_unit = data.get('stripe_unit', None)
+        stripe_count = data.get('stripe_count', None)
+        data_pool = data.get('data_pool', None)
+
+        # Set order
+        order = None
+        if obj_size > 0:
+            order = int(round(math.log(float(obj_size), 2)))
+
+        # Set features
+        feature_bitmask = self._format_features(features) if len(features) > 0 \
+            else 61  # FIXME: hardcoded int
```

Comment:
please remove this hardcoded value. If the user does not choose any feature then pass `feature_bitmask = None`, and librbd will take care of setting the default image features.

Reply:
Yes, will do